### ISSUE TITLE
Handle expired sessions properly?

### ISSUE THREAD

**Issue by vaijab** [UKHomeOffice/brp_app#68]
I was going through a user journey and left a page open midway. When I came back and did a refresh on that page, I got the following error:

```
Error: Session expired
   at checkSession (/app/node_modules/hmpo-form-wizard/lib/middleware/check-session.js:5:27)
   at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)
   at next (/app/node_modules/express/lib/router/route.js:131:13)
   at /app/node_modules/hmpo-form-wizard/lib/middleware/session-model.js:9:9
   at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)
   at next (/app/node_modules/express/lib/router/route.js:131:13)
   at Route.dispatch (/app/node_modules/express/lib/router/route.js:112:3)
   at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)
   at /app/node_modules/express/lib/router/index.js:277:22
   at Function.process_params (/app/node_modules/express/lib/router/index.js:330:12)
```

**Comment by easternbloc** [comment]
@lennym @gavboulton how do you handle session expired scenarios at the moment? Would be nice to use the same template if available...?

**Comment by lennym** [comment]
We don't have single template for this, as there are a number of different contexts in which the path out of an expired session differs, so we have then a number of child-router-level error handlers that set the template as a property on the error and then pass through to the top-level error handler to render.

Example:

In a child router:

```javascript
if (err.code === 'SESSION_TIMEOUT') {
  err.template = path.resolve(req.app.get('views'), './errors/session');
}
next(err);
```

In top level error handler: 

```javascript
err.template = err.template || 'errors/index';
res.render(err.template, {...});
```

The templates themselves are generally just a heading, a line of text explaining the situation and a link back to the start of the journey as appropriate, but they differ from journey to journey.

**Comment by JoeChapman** [comment]
Handled in our app with custom errorHandler

**Comment by easternbloc** [comment]
Fixed by #123
